templates/around/index.json

```diff
@@ -7,7 +7,7 @@
       "block_order": [],
       "settings": {
         "button_label": "",
-        "button_url": "/collections/all",
+        "button_url": "booqable://product/",
```

Comment:
isn't it plural `booqable://products` ?

Reply:
This one is a special shorthand and can be both singular and plural. It will redirect to dynamic collection. See `app/lib/shop_next/drops/shared/url_drop.rb`